doc/source/appendixb.rst

```diff
@@ -1272,6 +1304,31 @@ The Stromgren *uvby* throughputs are taken from
 >>> bp = S.ObsBandpass('stromgren,y')
 
 
+.. _pysynphot-nonhst-wfirst:
+
+WFIRST
+------
+
+Phase B estimates of the WFIRST integrated system throughputs have been taken from
+the `WFIRST Reference Information <https://wfirst.gsfc.nasa.gov/science/WFIRST_Reference_Information.html>`_ page at GSFC. 
+for more information. For example:
+
+>>> bp = S.ObsBandpass('wfirst,wfi,f062')
+
+Only the Wide Field Instrument (WFI) is currently supported with the following modes:
+
++------------+-----------------------------------------+
+|Description |Keywords                                 |
++============+=========================================+
+|Filter      |f062, f087, f106, f129, f146, f158, f184 |
++------------+-----------------------------------------+
+|Grating     |grism, prism                             |
++------------+-----------------------------------------+
+
+At this time, the estimated throughputs do not differentiate between the different sensor chip assemblies (SCAs).
+SCA-dependent throughputs will be delivered at a later time.
```

Comment:
Maybe something less obscure than "later time"? Jan 2021 or ...?

Reply:
I would normally agree, but I don't know when we might have another update. Could be in a year, could be in two, so I think it might be best to just stay vague on this one.